ee-feature-pack/galleon-shared/src/main/resources/modules/system/layers/base/org/wildfly/clustering/faces/mojarra/main/module.xml

```diff
@@ -33,7 +33,7 @@
     </resources>
 
     <dependencies>
-        <module name="jakarta.faces"/>
+        <module name="jakarta.faces.impl"/>
         <module name="jakarta.faces.api"/>
```

Comment:
if order matter, shouldn't api be before impl?

Reply:
I don't think order matters here. @bstansberry ?